.env.example

```diff
@@ -1,8 +1,14 @@
-AIRTABLE_BASE_ID='appYfW7a2loPD26Vg'
+## This AIRTABLE_BASE_ID is for the [Dev] Airtable base. 
+## If you want to access the [Prod] base, you will need to change it. AIRTABLE_BASE_ID='appYfW7a2loPD26Vg'
 
 ## Named to conform to airtable-schema-generator's expectations
 REACT_APP_AIRTABLE_API_KEY='YOUR-API-KEY-HERE'
 
 ## Needed for airtable-schema-generator if using auto mode
 AIRTABLE_EMAIL='YOUR-EMAIL-HERE'
-AIRTABLE_PASSWORD='YOUR-PASSWORD-HERE'
\ No newline at end of file
+AIRTABLE_PASSWORD='YOUR-PASSWORD-HERE'
+
+## For Sentry logging
+SENTRY_ORG='calblueprint'
+SENTRY_PROJECT='dccentralkitchen'
```

Comment:
This is the wrong project. Is this info actually used anywhere? It doesn't seem like it is.

Reply:
not sure--probably not? It was something that was included in the setup tutorial